src/commands/certs/issue.ts

```diff
@@ -239,10 +239,25 @@ export default async function issue(
     );
     return 1;
   }
-  if (cert instanceof ERRORS.DomainNotFound) {
+  if (
+    cert instanceof ERRORS.DomainNotFound ||
+    cert instanceof ERRORS.UnauthorizedCertsRequestError
+  ) {
     output.error(cert.message);
     return 1;
   }
+  if (cert instanceof ERRORS.CertsDNSError) {
+    output.error(
+      `We could not solve the ${cert.meta.type} challenge for cns ${
```

Comment:
lets replace `cert.meta.type` with `dns-01` because we know this will always be the error type associated with the `DNSError`